mlflow/server/js/src/gateway/components/api-keys/DeleteApiKeyModal.tsx

```diff
@@ -25,6 +26,58 @@ export const DeleteApiKeyModal = ({ open, secret, modelDefinitions, onClose, onS
 
   if (!secret) return null;
 
+  const renderEndpointsList = () => {
```

Comment:
Is this related to the security banner?

Reply:
No, these are additions to the statement when deleting either keys or endpoints to show additional information (bug bash feedback request)